README.md

```diff
@@ -5,16 +5,16 @@ Right now this software is clasiffied as a garbage. A lot of improvements are ne
 
 Main requirements:
 - cleaning not needed channels (with prior quarantine time) - done
-- monitoring time of users spent on each channel
+- monitoring time of users spent on each channel - done
```

Comment:
This is readme not a task board. Threat it as a documentation, not as a backlog

Reply:
fixed